examples/megacli.cpp

```diff
@@ -2500,6 +2500,35 @@ void exec_treecompare(autocomplete::ACState& s);
 void exec_querytransferquota(autocomplete::ACState& s);
 
 
+void exec_showattributes(autocomplete::ACState& s)
+{
+    if (Node* n = nodeFromRemotePath(s.words[1].s))
+    {
+        for (auto pair : n->attrs.map)
+        {
+            char namebuf[10]{};
+            AttrMap::nameid2string(pair.first, namebuf);
+            if (pair.first == 'c')
```

Comment:
Can we add a comment as to what `c` means?

Reply:
The comment would just say that 'c' is the index of the FileFingerprint which is made plain on the very next line, in code.

Reply:
In the future the code below  might move elsewhere and line 2511 might be isolated and it's not clear what it means.